raisinbread/RB_files/RB_MRICandidateErrors.sql

```diff
@@ -1,29 +1,5 @@
 SET FOREIGN_KEY_CHECKS=0;
 TRUNCATE TABLE `MRICandidateErrors`;
 LOCK TABLES `MRICandidateErrors` WRITE;
-INSERT INTO `MRICandidateErrors` (`ID`, `TimeRun`, `SeriesUID`, `TarchiveID`, `MincFile`, `PatientName`, `Reason`) VALUES (1,'2016-07-27 18:33:24','1.3.12.2.1107.5.2.32.35177.30000011091713000714000003150',19,'/tmp/TarLoad-10-32-GaHdaJ/rom162_400162_v6_20110917_011745_3_mri.mnc','ROM162_400162_V6','Visit label does not exist');
```

Comment:
Note: the imaging pipeline does not insert anything anymore in this table, which is why it is empty.

Reply:
where does this class of errors get logged now?

Reply:
When uploading manually, it does not let you upload scans with different PSCID-DCCID.

When uploading using batch_upload_imagingupload.pl, you get an error in the log of the upload saying the PSCID & DCCID does not match (saved in the notification spool table I think).